scripts/common/rook.sh

```diff
@@ -67,13 +67,10 @@ function remove_rook_ceph() {
     # make sure there aren't any PVs using rook before deleting it
     all_pv_drivers="$(kubectl get pv -o=jsonpath='{.items[*].spec.csi.driver}')"
     if echo "$all_pv_drivers" | grep "rook" &>/dev/null ; then
-        # do stuff
-        printf "%b" "$RED"
-        printf "ERROR: \n"
-        printf "There are still PVs using rook-ceph.\n"
-        printf "Remove these PVs before continuing.\n"
-        printf "%b" "$NC"
-        exit 1
+        logFail "There are still PVs using rook-ceph."
+        logFail "Remove these PV(s) before continuing:"
+        logFail "$all_pv_drivers"
+        return 1
```

Comment:
I don't think that this log matches the description
1. you should be filtering by `rook`
2. this is the PV _driver_, not the PV name, and that's not useful to the user

Reply:
@laverya good point. Changed. WDYT? Can we move now?